background/lib/nfts.ts

```diff
@@ -0,0 +1,88 @@
+import { AddressOnNetwork } from "../accounts"
+import { EVMNetwork } from "../networks"
+import { NETWORK_BY_CHAIN_ID } from "../constants"
+import { getNFTs as alchemyGetNFTs, AlchemyNFTItem } from "./alchemy"
+import { getNFTsByOwners, SimpleHashNFTModel } from "./simple-hash"
```

Comment:
What do you think about keeping names the same?
```suggestion
import { getNFTs as simpleHashGetNFTs, SimpleHashNFTModel } from "./simple-hash"
```

Reply:
I like it, though it makes me think the function signatures should be more similar as well.

Reply:
🤩